.rubocop.yml

```diff
@@ -681,3 +682,138 @@ Style/IfUnlessModifierOfIfUnless:
 Style/ZeroLengthPredicate:
   Enabled: false
 
+# DISABLED
+Bundler/OrderedGems:
+  Enabled: false
+
+# DISABLED
+Layout/EmptyLineAfterMagicComment:
+  Enabled: false
+
+# DISABLED
+Layout/EmptyLinesAroundBeginBody:
+  Enabled: false
+
+# DISABLED
+Layout/EmptyLinesAroundExceptionHandlingKeywords:
+  Enabled: false
+
+# DISABLED
+Layout/IndentHeredoc:
+  Enabled: false
+
+# DISABLED
+Layout/MultilineArrayBraceLayout:
+  Enabled: false
+
+# DISABLED
+Layout/MultilineHashBraceLayout:
+  Enabled: false
+
+# DISABLED
+Layout/MultilineMethodCallBraceLayout:
+  Enabled: false
+
+# DISABLED
+Layout/MultilineMethodDefinitionBraceLayout:
+  Enabled: false
+
+# DISABLED
+Layout/SpaceInsidePercentLiteralDelimiters:
+  Enabled: false
+
+# DISABLED - TODO PUP-7948
+Lint/EmptyExpression:
+  Enabled: false
+
+# DISABLED
+Lint/EmptyWhen:
+  Enabled: false
+
+# DISABLED
```

Comment:
I don't think I was clear in my [earlier comment](https://github.com/puppetlabs/puppet/pull/6245#discussion_r143020663). I would remove the bare `# DISABLED` comment as well.